lib/Conversion/TorchToMhlo/GatherOp.cpp

```diff
@@ -109,11 +109,12 @@ class ConvertAtenOp : public OpConversionPattern<AtenOpT> {
                   ConversionPatternRewriter &rewriter) const override;
 };
 
-// Ref: https://pytorch.org/docs/stable/generated/torch.nn.functional.embedding.html
+// Ref:
+// https://pytorch.org/docs/stable/generated/torch.nn.functional.embedding.html
 // padding_idx (int, optional)
-//  – If specified, the entries at padding_idx do not contribute to the gradient;
-//  therefore, the embedding vector at padding_idx is not updated during training,
-//  i.e. it remains as a fixed “pad”.
+//  – If specified, the entries at padding_idx do not contribute to the
```

Comment:
The changes in this file are not needed, right?

Reply:
same I think these came when I ran clang format...